### PR TITLE
Accessibility: Page Head

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://salsadigital.atlassian.net/browse/WSM-188

### Changed

- Implement a store to set head.
- Include "| site name" to title.
- Updated the following pages to use the store value:
  - Search pages (including /search, /whatson, /grants, /profiles)
  - Authenticated pages (login / reset)
  - Publication PrintAll
  - Sitemap
  - Tide pages (landing pages, 400 / 500 pages, and generally all other pages)

### Screenshots

Screen Shot 2020-04-17 at 11 00 30 am
Screen Shot 2020-04-17 at 11 00 05 am
Screen Shot 2020-04-17 at 10 59 56 am
Screen Shot 2020-04-17 at 10 59 40 am
Screen Shot 2020-04-17 at 10 59 20 am

### CODE REVIEW

**Comment by tim-yao** [comment]
Hey @alan-cole I haven't review your PR yet, but do you think `titleTemplate` should be used in this task?https://nuxtjs.org/api/configuration-head/

**Comment by alan-cole** [comment]
Hi @tim-yao, That makes sense. I've implemented the titleTemplate in nuxt.config.js.

**Comment by alan-cole** [comment]
> Hi @alan-cole will this PR add breaking changes? I feel it is, may need you to do some checking for this. We have a lot customer sites which has their custom pages, they should still able to use original way to add `title` to `head`. Otherwise after this release, all custom pages title may won't work.

Thanks @tim-yao , I've had a check, creating a custom page in tide/modules with no head
```
<template>
  <rpl-page-layout><h1>My Example Page</h1></rpl-page-layout>
</template>

<script>
import { RplPageLayout } from '@dpc-sdp/ripple-layout'
export default {
  name: 'MyPage',
  components: { RplPageLayout }
}
</script>
```
 will result in:
no-head

With the title: "vic.gov.au | vic.gov.au"

Providing a head() in a component
```
<template>
  <rpl-page-layout><h1>My Example Page</h1></rpl-page-layout>
</template>

<script>
import { RplPageLayout } from '@dpc-sdp/ripple-layout'
export default {
  name: 'MyPage',
  components: { RplPageLayout },
  head () {
    return {
      title: 'Custom My Page Title - This should override the Veux Store Value'
    }
  }
}
</script>
```
will result in:
with-head

With the title: "Custom My Page Title - This should override the Veux Store Value | vic.gov.au"

File: packages/ripple-nuxt-tide/modules/publication/pages/PrintAll.vue [hunk 1]
```diff
@@ -87,11 +87,6 @@ export default {
   props: {
     sidebar: Boolean
   },
-  head () {
-    return {
-      title: this.publication.title
-    }
-  },
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/modules/publication/pages/PrintAll.vue:94]: I would say for all static pages, this is more straight forward and easy to understand(Nuxt default). Not really feel we need to change to use store. How do you think?

**Pull request merged**